Test-utils does not work with latest vitest

### Environment

- Operating System: `Darwin`
- Node Version:     `v16.16.0`
- Nuxt Version:     `3.0.0-rc.4`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `srcDir`, `vite`, `build`, `modules`, `tailwindcss`, `runtimeConfig`
- Runtime Modules:  `@nuxtjs/tailwindcss@5.1.3`, `nuxt-graphql-client@0.0.33`
- Build Modules:    `-`

### Reproduction

Run tests with test-utils and vitest >= 0.14.0

### Describe the bug

When using vitest newer than 0.13.1 the following error appears:

`Cannot find module './node_modules/vitest/dist/node.js' imported from ./node_modules/@nuxt/test-utils-edge/dist/index.mjs`

### Additional context

This has to do with this change in vitest 0.14.0: https://github.com/vitest-dev/vitest/commit/692ad34

### Logs

_No response_

--- a/packages/test-utils/src/run.ts
+++ b/packages/test-utils/src/run.ts
@@ -21,7 +21,7 @@ export async function runTests (opts: RunTestOptions) {
     process.env.NUXT_TEST_DEV = 'true'
   }
 
-  const { startVitest } = await import('vitest/dist/node.js')
+  const { startVitest } = await import('vitest/dist/node.mjs')
   const succeeded = await startVitest(
     [] /* argv */,
     // Vitest options

